metadata-ingestion/src/datahub/ingestion/source/dbt/dbt_core.py

```diff
@@ -162,7 +162,7 @@ def extract_dbt_entities(
             materialization = manifest_node["config"]["materialized"]
 
         upstream_nodes = []
-        if "depends_on" in manifest_node:
+        if "depends_on" in manifest_node and 'nodes' in manifest_node['depends_on']:
```

Comment:
Looks like the linter doesn't like the single quotes

```suggestion
        if "depends_on" in manifest_node and "nodes" in manifest_node["depends_on"]:
```